__tests__/app.test.js

```diff
@@ -85,16 +85,79 @@ describe("/api", () => {
         });
     });
 
-        test("GET 400: sends an appropriate status and error message when given an invalid id", () => {
-          return request(app)
-            .get("/api/articles/nonsense")
-            .expect(400)
-            .then((response) => {
-              expect(response.body.msg).toBe("Bad request");
-            });
+    test("GET 400: sends an appropriate status and error message when given an invalid id", () => {
+      return request(app)
+        .get("/api/articles/nonsense")
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe("Bad request");
         });
+    });
+  });
+
+  describe("GET /articles", () => {
+    test("GET 200: sends an array of article objects with following properties: author,title, article_id,topic, created_at, votes, article_img_url. Not including a body property ", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.length).toBe(data.articleData.length);
+          articles.forEach((article) => {
+            expect(typeof article.article_id).toBe("number");
+            expect(typeof article.author).toBe("string");
+            expect(typeof article.title).toBe("string");
+            expect(typeof article.topic).toBe("string");
+            expect(typeof article.created_at).toBe("string");
+            expect(typeof article.votes).toBe("number");
+            expect(typeof article.article_img_url).toBe("string");
+            expect(article.body).toBe(undefined)
+          });
+        });
+    });
+
+    test("GET 200: sends an array of article objects including a comment count property", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          
```

Comment:
a length check is needed otherwise all of the array methods can produce false positives :)